src/api/v4/leaflet-layer.js

```diff
@@ -88,6 +90,11 @@ var LeafletLayer = L.TileLayer.extend({
     this._triggerLayerFeatureEvent(Layer.events.FEATURE_OUT, internalEvent);
   },
 
+  _onFeatureError: function (error) {
+    var cartoError = new CartoError(error);
+    this.fire(Layer.events.FEATURE_ERROR, cartoError);
```

Comment:
IMO this is not really a feature error... If Maps API returns some errors when calculating the interactivity it might affect multiple features/layer, not only one feature/layer as the name implies. Perhaps we can find a better name?

This class is not firing any other CARTO events... Its the layer "model" who triggers featureClicked, featureOver, etc... Shouldn't we unify this (ie: either the model or the view fires this short of events).

Are we documenting this event?

Reply:
The problem in this case it's that the error is not attached to any particular layer. So we have three options:
- Trigger it from the layer (as it's in this PR).
- Trigger it from the first layer that has interactivity.
- Trigger it from all the layers that have interactivity.

For me, the second one is good enough and we can refactor this code to support it. The problem will affect that first layer so it's a good place to trigger the event.

@IagoLast was about to change the name to TILE_ERROR if I'm not mistaken.

Reply:
Any thoughts on this @alonsogarciapablo @IagoLast @Jesus89 ?

My vote if triggering from the first layer with interactivty.

Reply:
My vote -> `Trigger it from all the layers that have interactivity.`